src/lib/Microsoft.Health.Fhir.Ingest.Template/CalculatedFunction/CalculatedFunctionContentTemplate.cs

```diff
@@ -0,0 +1,45 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Microsoft.Health.Fhir.Ingest.Template.CalculatedFunction
```

Comment:
Another option is to simply put this in the template namespace.  Especially since you are core functionality that the objects in the Template namespace are leveraging.

Reply:
The reason I had moved this into its own namespace is that I felt that the `Template` namespace was getting pretty big, and didn't want to add to it with 20+ new classes. But you're right in that there is a strong coupling to the core functionality in the Template namespace. So I'll just get rid of `CalculatedFunction` namespace and move everything  into the `Template` namespace.